modules/tinymce/src/themes/silver/main/ts/ui/toolbar/button/ToolbarButtons.ts

```diff
@@ -328,7 +330,8 @@ const renderSplitButton = (spec: Toolbar.ToolbarSplitButton, sharedBackstage: Ui
     onItemExecute: (a, b, c) => { },
 
     splitDropdownBehaviours: Behaviour.derive([
-      DisablingConfigs.splitButton(false),
+      DisablingConfigs.splitButton(false || sharedBackstage.providers.isReadonly()),
```

Comment:
This doesn't make sense, why are you doing `false || sharedBackstage.providers.isReadonly()`? As false will never match so this could just be `sharedBackstage.providers.isReadonly()`.

- [ ] Remove `false` as it doesn't do anything